Etabs_Adapter/CRUD/Create/Material.cs

```diff
@@ -143,7 +158,130 @@ private eMatType MaterialTypeToCSI(MaterialType materialType)
 
         /***************************************************/
 
+        private bool ISetDesignMaterial(IMaterialFragment material)
+        {
+            return SetDesignMaterial(material as dynamic);
+        }
+
+        /***************************************************/
+
+        private bool SetDesignMaterial(Concrete material)
+        {
+            bool success = true;
+
+            double fc = 0;
+            bool lw = false;
+            double fcsFactor = 0;
+            int sstype = 0;
+            int sshystype = 0;
+            double strainAtFc = 0;
+            double strainUltimate = 0;
+            double finalSlope = 0;
+            double frictionAngle = 0;
+            double dilationAngle = 0;
+
+            m_model.PropMaterial.GetOConcrete_1(
+                material.DescriptionOrName(),
+                ref fc,
+                ref lw,
+                ref fcsFactor,
+                ref sstype,
+                ref sshystype,
+                ref strainAtFc,
+                ref strainUltimate,
+                ref finalSlope,
+                ref frictionAngle,
+                ref dilationAngle
+                );
+
+            success &= (0 == m_model.PropMaterial.SetOConcrete_1(
+                material.DescriptionOrName(),
+                material.CylinderStrength,
+                IsLightweight(material),
+                IsLightweight(material) ? 0.75 : 1.0, //Strength reduction factor for lightweight concrete
```

Comment:
the 0.75 here I presume is american code? Here I also think we need to be a bit carful, and might need to have a slight think on how we set this up properly.

Reply:
Agreed, will not set this property, and raise a note only if the concrete appears to be lightweight